src/packages/piral-oidc/src/types.ts

```diff
@@ -42,6 +42,24 @@ export interface OidcConfig {
    * Otherwise, the client is responsive to the `before-fetch` event.
    */
   restrict?: boolean;
+  /**
+   * If provided, the window will redirect to this Uri after getting
+   * a new session from the redirectUri callback.
+   */
+  appUri?: string;
+  /**
+   * If provided, logging will be enabled for the oidc-client.
+   * Defaults to Log.DEBUG in development NODE_ENV.
+   */
+  logLevel?: LogLevel;
+}
+
+export enum LogLevel {
```

Comment:
I would advise for a `const enum`. Less runtime code and also allows just constants such as `0` for non-TS users.

Regarding the member names we usually use standard camelCase, e.g., 

```ts
export const enum LogLevel {
  none,
  error,
  warn,
  info,
  debug,
};
```

Reply:
After speaking on gitter, I think we will use 
```typescript
const enum LogLevel {
  none = 'none',
  error = 'error',
...etc
}
```

Reply:
Completed this